release/nightly_tests/dataset/data_ingest_benchmark.py

```diff
@@ -10,23 +10,43 @@
 from ray.data import DatasetPipeline
 
 import pandas as pd
+import torch
 
 GiB = 1024 * 1024 * 1024
 
 
-@ray.remote(num_cpus=0.5)
+@ray.remote
 class ConsumingActor:
     def __init__(self, rank):
         self._rank = rank
-
-    def consume(self, split):
-        DoConsume(split, self._rank)
+        self._use_gpu = use_gpu
+
+    def consume(
+        self,
+        split,
+        expected_total_read_bytes,
+        use_gpu=False,
+        early_stop_when_read_bytes=None,
+    ):
+        do_consume(
+            split,
+            self._rank,
+            expected_total_read_bytes,
+            use_gpu,
+            early_stop_when_read_bytes,
+        )
 
     def get_location(self):
         return ray.get_runtime_context().get_node_id()
 
 
-def DoConsume(split, rank):
+def do_consume(
+    split,
+    rank,
+    expected_total_read_bytes,
+    use_gpu=False,
+    early_stop_when_read_bytes=None,
```

Comment:
Nit: this variable reads like a bool to me. Is there another name we could use? If not, I think it'd help to add a type hint.

Reply:
Good point. I'lll rename it to something like `max_bytes_to_read`.